_posts/2021-05-21-pytesting-in-openfaas.md

```diff
@@ -0,0 +1,331 @@
+---
+title: "Testing Python Functions with Pytest"
```

Comment:
Testing Python Functions with Pytest <- may lead to people landing here looking for regular Python testing, so could include openfaas or serverless or some variation.